modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+= TinyMCE 6.4
+:navtitle: TinyMCE 6.4
+:description: Release notes for TinyMCE 6.4
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.4 was released for {enterpriseversion} and {cloudname} on Wednesday, March 15^th^, 2023. It includes {productname} 6.4, additional changes to Premium plugins, and a new Premium plugin, Advanced Template. These release notes provide an overview of the changes for {productname} 6.4:
+
+* xref:new-premium-plugin[New Premium Plugin]
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-plugin-end-of-life-announcement[Accompanying Premium Plugin end-of-life announcement]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvement[Improvement]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+// * xref:security-fixes[Security fixes] -- leave until closer to 2023-03-15 and remove or make visible as appropriate.
+// * xref:known-issues[Known issues] -- leave until closer to 2023-03-15 and remove or make visible as appropriate.
+// NB: edit the equivalent section of nav.adoc when the remove or make visible decision is made.
+
+[[new-premium-plugin]]
+== New Premium Plugin
+
+=== Advanced Template 1.1.0
+
+The {productname} 6.4 release includes the release of a new premium plugin: **Advanced Templates**.
+
+The **Advanced Template** Premium plugin enables the creation of complex, interactive templates using {productname}.
+
+Templates are declared as `const` variables that can be setup as part of a {productname} instance.
+
+They can also be added to a {productname} instance by end-users selecting content in a {productname} document and saving the selection as a template via the {productname} user-interface.
+
+For information on the **Advanced Template** plugin, see: xref:advanced-template.adoc[Advanced Templates].
```

Comment:
```suggestion
=== Advanced Template 1.1.0

The {productname} 6.4 release includes the release of a new premium plugin: **Advanced Template**.

The **Advanced Template** Premium plugin enables the creation of complex, interactive templates using {productname}.

Templates are declared as `const` variables that can be setup as part of a {productname} instance.

They can also be added to a {productname} instance by end-users selecting content in a {productname} document and saving the selection as a template via the {productname} user-interface.

For information on the **Advanced Template** plugin, see: xref:advanced-templates.adoc[Advanced Template].
```
The plugin name is Advanced Template, however the working adoc file is `advanced-templates.adoc`, which cannot be changed as it is already referenced in the help dialog in 6.4